newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -114,7 +114,6 @@ const SortableList = SortableContainer(
               <ImageThumbnail
                 key={sprite.ptr}
                 selectable
-                key={sprite.ptr}

```

Comment:
🙏

Reply:
haha yeah, looks like we had the same idea but the merge didn't really work out 😓